src/dialogs/stat_dialogs/chart_options_model.pas

```diff
@@ -0,0 +1,144 @@
+unit chart_options_model;
+
+{$mode objfpc}{$H+}
+
+interface
+
+uses
+  Classes, SysUtils, stat_dialog_contribution, chart_options, executor,
+  epidatafiles;
+
+const
+  ScatterMinMax = %1111;
```

Comment:
This "magic" value is really hard to read and understand. I can see that it is used in the option view as a marker for whether the input field supports dates, but it is really complicated to read and understand.
I would prefer proper boolean fields with easy to understand names that can be used in simple if statements. It gives a much better readability and in the future we don't fumble to understand what this magic value does.

Reply:
Done